src/status_im/ui/components/toolbar/view.cljs

```diff
@@ -79,12 +79,15 @@
 (defn content-title
   ([title] (content-title nil title))
   ([title-style title]
-   (content-title title-style title nil nil))
-  ([title-style title subtitle-style subtitle]
+   (content-title title-style title nil nil nil))
+  ([title-style title subtitle-style]
```

Comment:
`subtitle` missed here ,`[title-style title subtitle-style subtitle]`

Reply:
inadvertence :-1:

Reply:
@tbenr any updates on this?

Reply:
@flexsurfer should be ok now.